tests/snapshots/index.js.md

```diff
@@ -214,7 +230,7 @@ Generated by [AVA](https://avajs.dev).
 
 > stdout
 
-    'Token revoked'
+    '::warning::Token revocation failed: Invalid URL'
```

Comment:
This and the above change are incorrect. Something is wrong with how the URL for the request is set, the request should not fail. It's also the reason why coverage dropped below 100%

Reply:
Resolved by c1f802e.